README.md

```diff
@@ -19,19 +19,19 @@ compliant with HTML5 and they will look OK.
 
 Details are here: [yegor256.github.io/tacit](https://yegor256.github.io/tacit/)
 
-Just add it to your HTML (replace `VERSION` with the latest version number,
-which you can find [here](https://github.com/yegor256/tacit/releases)):
+ Just add it to your HTML <!-- (replace `VERSION` with the latest version number,
+which you can find [here](https://github.com/yegor256/tacit/releases)): -->
 
 ```html
 <!DOCTYPE html>
 <html>
   <head>
-    <link rel="stylesheet" href="https://cdn.rawgit.com/yegor256/tacit/gh-pages/tacit-css-VERSION.min.css"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/yegor256/tacit@1.3.6/dist/tacit-css-1.3.6.min.css"/>
   </head>
 </html>
 ```
 
-Or simply download [`tacit-css.min.css`](https://raw.githubusercontent.com/yegor256/tacit/gh-pages/tacit-css.min.css)
```

Comment:
@Dontmindeme please revert this line back to it's original state

The url in this case should be:
```
https://raw.githubusercontent.com/yegor256/tacit/gh-pages/tacit-css.min.css
```

Reply:
@Dontmindeme forget this one comment, please change the url to:
```https://cdn.jsdelivr.net/gh/yegor256/tacit@gh-pages/tacit-css.min.css```

Reply:
...